scripts/populate_database.py

```
@@ -698,6 +702,124 @@ async def fetch_game_simple(app_id: str) -> bool:
+                failed_count += len(chunk)
+                if TQDM_AVAILABLE:
+                    pbar.update(len(chunk))
+
+        if TQDM_AVAILABLE:
+            pbar.close()
+
+        # Cache new mappings
+        if new_mappings:
+            bulk_cache_itad_mappings(new_mappings)
+
+        print(f"\n🔗 ITAD ID resolution complete!")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)